mixins/localize-mixin.md

```diff
@@ -56,27 +58,56 @@ class MyComponent extends LocalizeStaticMixin(LitElement) {
 
 }
 ```
-
 #### Example 2: Dynamically Imported Resources
 
 This approach is better for components with many language resources. By importing them dynamically, only the resources for the requested language are actually fetched and downloaded.
 
 Store your resources in individual files, one for each language:
 ```javascript
 // en.js
-export const val = {
+export default {
   'hello': 'Hello {firstName}!'
 };
 ```
 
-Then dynamically import the matching file:
+**Note:** To avoid accidental imports and errors, your resource files should have a dedicated directory with nothing else in it.
+
+Then create your `config` getter:
+```javascript
+import { LocalizeDynamicMixin } from '@brightspace-ui/core/mixins/localize-dynamic-mixin.js';
+
+class MyComponent extends LocalizeDynamicMixin(LitElement) {
+
+  static get config() {
+    return {
+      importFunc: lang => import(`../lang/${lang}.js`) // Path must be relative!
+    };
+  }
+}
+```
+
+Or with additional optional properties in the `config` object:
+
+```javascript
+return {
+  ...,
+  osloCollection: 'my-project\\myComponent', // To enable OSLO
+  exportName: 'resources' // If your resource files use named exports
+}
+```
+
+#### Example 3: Manually Retrieved Resources
```

Comment:
We could even ditch this section from the docs altogether if you want. 🤷

Reply:
I thought about it, but if people use this outside BSI it could be helpful.